cirq-core/cirq/ops/util.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2022 The Cirq Developers
```

Comment:
nit: Can we give a better name to this module? util is too generic and it'll be hard to guess what's inside it by looking at the name. Maybe `ops/qid_util.py` ?

Reply:
Done.